src/helpers.ts

```diff
@@ -25,14 +25,14 @@ export function int(value: BigNumber): Integer {
   return value.toString();
 }
 
-export function chunkArray<T>(array: T[], size: number) {
+export function chunkArray<T>(array: T[], size: number): T[][] {
   if (size < 1) {
     throw new Error(`Size needs to be positive: ${size}`);
   }
 
-  let result = [];
+  const result = [];
```

Comment:
Now we get warnings for `const` when there's no reassignment

Reply:
nice